pkg/kubelet/status_manager.go

```diff
@@ -58,6 +60,8 @@ func (s *statusManager) Start() {
 		err := s.syncBatch()
 		if err != nil {
 			glog.Warningf("Failed to updated pod status: %v", err)
+			// Errors and tight-looping are bad, m-kay
+			time.Sleep(30 * time.Second)
```

Comment:
This is introduced in @brendandburns original commits, and you are right 30s is too long, and it is irrelevant to this pr. My bad ignoring it. I am going to remove it for now.